### PR TITLE
docs: Update Loki components page with new bloom components

### PR DESCRIPTION
**Pull request by chaudum**
**What this PR does / why we need it**:

The page was outdated and did not mention the new bloom planner and bloom builder components, but still mentioned the bloom compactor.

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [x] Documentation added
- [ ] Tests updated
- [x] Title matches the required conventional commits format, see [here](https://www.conventionalcommits.org/en/v1.0.0/)
  - **Note** that Promtail is considered to be feature complete, and future development for logs collection will be in [Grafana Alloy](https://github.com/grafana/alloy). As such, `feat` PRs are unlikely to be accepted unless a case can be made for the feature actually being a bug fix to existing behavior.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

Signed-off-by: Christian Haudum <christian.haudum@gmail.com>

### CODE REVIEW

File: docs/sources/get-started/components.md [hunk 1]
```diff
@@ -337,18 +338,28 @@ from the query frontend.
...
 
-The Bloom Compactor service is responsible for building blooms for chunks in the object store. 
+The Bloom Planner service is responsible for planning the tasks for blooms creation. It runs as a singleton and provides a queue
+from which tasks are pulled by the Bloom Builders. The planning runs periodically and takes into account what blooms have already
+been built for a given day and tenant and what series need to be newly added.
+
+This service is also used to apply blooms retention.
+
+## Bloom Builder
+{{% admonition type="warning" %}}
```
Comment (JStickler) [inline docs/sources/get-started/components.md:353]:
```suggestion
{{< admonition type="warning" >}}
```

File: docs/sources/get-started/components.md [hunk 2]
```diff
@@ -337,18 +338,28 @@ from the query frontend.
...
+This feature is an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available.  No SLA is provided.  
+{{% /admonition %}}
```
Comment (JStickler) [inline docs/sources/get-started/components.md:355]:
```suggestion
{{< /admonition >}}
```

**Review by JStickler — COMMENTED** [review]
[docs team]  LGTM

**Comment by chaudum** [comment]
@JStickler I removed the backport label for 3.0.x and 3.1.x, because these components are new.

**Review by JStickler — APPROVED** [review]

**Pull request merged**